src/components/user/SignUpForm/index.js

```diff
@@ -6,23 +6,28 @@ import { View, Text } from 'react-native';
 import * as constraints from 'utils/constraints';
 import Input from 'components/common/Input';
 import Button from 'components/common/Button';
+import PickerScreen from 'components/common/PickerScreen';
 import translate from 'utils/i18n';
 import styles from './styles';
 import { whiteColor } from '../../../constants/styleConstants';
 import { screens } from '../../../constants/screenConstants';
 
 const SignUpForm = ({ handleSubmit, error, navigator }) => {
-  const onPress = () => {
+  const redirectToSignIn = () => {
     navigator.push({
-      screen: screens.signUp
+      screen: screens.login
     });
   };
 
+  const genders = ['SELECT YOUR GENDER', 'male', 'female'];
+
+  const preSelected = 'SELECT YOUR GENDER';
```

Comment:
Does the picker have a placeholder? If it does perhaps it's a better way to handle this. If not or does not look good I am ok with leaving it like this

Reply:
it does not have a placeholder. I've checked the documentation and I didn't find one

Reply:
But I agree it is not the best way to handle it